src/incubator/TabController/index.js

```diff
@@ -71,18 +80,31 @@ class TabController extends Component {
 
   render() {
     const {itemStates, ignoredItems} = this.state;
+    
+    // Rounding on Android, cause it cause issues when comparing values
+    const screenWidth = Constants.isAndroid ? Math.round(Constants.screenWidth) : Constants.screenWidth;
```

Comment:
Is landscape support is on the todo list?

Reply:
yes, it won't work on Landscape at the moment I'm aware.